### PR TITLE
refine/update webxdc docs

### PR DESCRIPTION
**Pull request by hpk42**
- talk about messengers and not Delta Chat (in particular not about the ancient 1.38) 
- Rewrite Intro to mention and reference more content, and describe the basic "HTTP -> P2P" move 
- more prominently link webxdc-dev tool for development

### CODE REVIEW

**Comment by hpk42** [comment]
preview is here: https://merlinux.eu/webxdc-new/book/intro.html

**Review by r10s — APPROVED** [review]
lgtm, also build locally and also did not find typos :)

File: src/intro.md [hunk 1]
```diff
@@ -1,14 +1,67 @@
...
-Webxdc is a new way to create and share web apps in messenger chat groups. Anyone can build HTML5 apps, package it as a [`.xdc` file](./spec/index.html#webxdc-file-format), and drop it in a chat to share with friends. Once shared, chat participants can simply click "Start" to run the app and interact with each other through the app.
+Webxdc is a P2P oriented container format for sharing and hosting HTML5 web apps in messenger chats. 
+The e-mail based [Delta Chat](https://delta.chat) and the XMPP-based [Cheogram](https://cheogram.com) messengers 
+support [webxdc apps](https://webxdc.org/apps), which run on both messengers without any change. 
+
+Instead of being hosted on a central always-online HTTP server, 
+webxdc apps are stored with the host messenger by attaching a container file to a chat message. 
+Instead of using the HTTP protocol for querying a central server to obtain authoritative application state,
+webxdc apps send and receive messages via the messenger's [JavaScript API]
+which allows to send and receive "application updates" between devices particpating in a chat.
```
Comment (r10s) [inline src/intro.md:11]:
```suggestion
which allows to send and receive "application updates" between devices participating in a chat. 
```

File: src/intro.md [hunk 2]
```diff
@@ -1,14 +1,67 @@
...
+App developers do not need to implement message transport, user authentication, 
+discovery or bootstrapping mechanisms, and they do not need to implement end-to-end encryption either. 
+All Authentication, identity management, social discovery and message transport 
+is outsourced to the host messenger which runs webxdc apps,
+thereby letting each app inherit offline-first and end-to-end encryption 
+capabilities of the hosting messenger. 
+
+Messengers run webxdc container files in network-isolated webviews that can not perform any DNS or HTTP queries.
+Webxdc apps can only cause network messages by calling the [`sendUpdate`](./spec/sendUpdate.html)
+and [`setUpdateListener`](./spec/setUpdateListener.html) Javasript APIs,
```
Comment (r10s) [inline src/intro.md:21]:
```suggestion
and [`setUpdateListener`](./spec/setUpdateListener.html) JavaScript APIs,
```

File: src/intro.md [hunk 3]
```diff
@@ -1,14 +1,67 @@
...
+implemented by the hosting messenger. 
+In ["Bringing E2E privacy to the web"](https://delta.chat/en/2023-05-22-webxdc-security) 
+Delta Chat developers described the unqiue security-audited privacy guarantees of webxdc
```
Comment (r10s) [inline src/intro.md:24]:
```suggestion
Delta Chat developers described the unique security-audited privacy guarantees of webxdc 
```

File: src/intro.md [hunk 4]
```diff
@@ -1,14 +1,67 @@
...
+
+Webxdc app development and deployment is fundamentally easier than 
+developing for and maintaining an application-specific always-online HTTP server. 
+But there are undeniably complications in arranging consistent web app state 
+across user's devices, a typical issue for any P2P system. 
+The [Shared web application state] chapter 
+provides useful theoretical and practical background for writing robust offline-first web apps
+without requiring a central online authority acting as the "source of truth". 
+Even if you don't study the topic in depth, reading [Shared web application state] 
+introduces you to the terminology and neccessary considerations
```
Comment (r10s) [inline src/intro.md:53]:
```suggestion
introduces you to the terminology and necessary considerations 
```

**Review by r10s — COMMENTED** [review]
found some typos, other wise still lgtm (even better :)

**Comment by hpk42** [comment]
thanks -- i am merging this now.

**Pull request merged**